server/routes/findInterventions/interventionsFilter.ts

```diff
@@ -0,0 +1,46 @@
+import { Request } from 'express'
+import { InterventionsFilterParams } from '../../services/interventionsService'
+
+export default class InterventionsFilter {
+  pccRegionIds: string[] | undefined
+
+  gender: ('male' | 'female')[] | undefined
```

Comment:
No biggie (and something we can do after this is merged if you think it's a good idea), but could be worth storing these in an exported enum, so we could use them all over the codebase?

Reply:
It's not yet clear to me what the enum would represent. It's not "gender", it's just "the options that we display under ‘gender’ on the filter list". This type is quite specific to that particular set of filter options. If it becomes clear that those filter options need to be representable outside of one screen or something, then we can think about extracting them.

Reply:
Sorry, when I say "all over the codebase" I think I probably just meant this file (I saw a lot of 'male' and 'female' in the tests, so thought it'd make it more standardised, but it's probably not right to use this in the tests). Probably overkill, and you're right, it doesn't really represent "gender" itself.